changelog/665.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+secret/pki: Delete invalid certificates during tidy if they cannot be parsed due to Go's x509 handling  and `tidy_invalid_certs=true`.
```

Comment:
nit:

```suggestion
secret/pki: Delete invalid certificates during tidy via `tidy_invalid_certs=true` if they cannot be parsed due to Go's x509 handling.
```

Reply:
I'll add all these suggestions in one commit with the test :D Thank you!